### PR TITLE
[mon] Improve monitoring interface

### PR DESCRIPTION
**Pull request by kostorr**
Closes ORC-330

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -417,9 +417,9 @@ metric format for the CRORC and the CRU is different, as different parameters ar
 | Value name             | Value                    | Type   |
 | ---------------------- | ------------------------ | ------ |
 | `"pciAddress"`         | -                        | string |
-| `"qsfp"`               | "Enabled" or "Disabled"  | string |
-| `"offset"`             | "Dynamic" or "Fixed"     | string |
-| `"timeFrameDetection"` | "Enabled" or "Disabled"  | string |
+| `"qsfp"`               | 0/1 (Disabled/Enabled)   | int    |
+| `"dynamicOffset"`      | 0/1 (Disabled/Enabled)   | int    |
```
Comment (kostorr) [inline README.md:421]: Please note this change.

File: README.md [hunk 2]
```diff
@@ -429,27 +429,28 @@ metric format for the CRORC and the CRU is different, as different parameters ar
...
-| -------------- | ----------------------- | ------ | 
-| `"pciAddress"` | -                       | string |
-| `"clock"`      | "TTC" or "local"        | string |
-| `"offset"`     | "Dynamic" or "Fixed"    | string |
-| `"userLogic"`  | "Enabled" or "Disabled" | string |
+| Value name        | Value                   | Type   | 
+| ----------------- | ----------------------- | ------ | 
+| `"pciAddress"`    | -                       | string |
+| `"clock"`         | "TTC" or "Local"        | string |
+| `"dynamicOffset"` | 0/1 (Disabled/Enabled)  | int    |
```
Comment (kostorr) [inline README.md:452]: And this.

**Comment by awegrzyn** [comment]
So I review README and it looks good to me.
To verify it 100% I need an RPM, do you want me to generate one for this branch (this will take more time), or do you prefer to merge, release and the I'll trigger the RPM build?

**Comment by kostorr** [comment]
I'll release a new version and RPM. Will ping you when it's out.

**Pull request merged**